### PR TITLE
[11.0] mark invoice as paid by transactions

### PR DESCRIPTION
**Pull request by robinkeunen**
[task](https://gestion.coopiteasy.be/web?#id=6045&view_type=form&model=project.task&action=475&active_id=165)

I overloaded `_get_payment_vals` function from account.invoice to pass it to the report. From my investigations, it should be harmless because that function and other functions calling it are used only in imports and frontend. BUT I'd love a second look at that from someone else ( @vvrossem )

### CODE REVIEW

File: account_invoice_paid_transaction/demo/model_name_demo.xml [hunk 1]
```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- Copyright <YEAR(S)> <AUTHOR(S)>
+     License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl). -->
+
+<odoo>
+
+<record id="model_name_my_record_demo" model="model.name">
+    <!-- My demo record data -->
```
Comment (vvrossem) [inline account_invoice_paid_transaction/demo/model_name_demo.xml]: WIP ?

File: account_invoice_paid_transaction/readme/DESCRIPTION.rst [hunk 2]
```diff
@@ -0,0 +1,2 @@
+Mark invoice as paid if an done transactions exists for the origin sale order.
```
Comment (vvrossem) [inline account_invoice_paid_transaction/readme/DESCRIPTION.rst]: typo `any`

File: account_invoice_paid_transaction/views/report_name.xml [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+    report_type="qweb-pdf"
+    groups="user_group"
+    attachment="object.state == 'done' and 'something.pdf'"
+    multi="False"/>
+
+<record id="name_action" model="ir.actions.report.xml">
+    <field name="paperformat_id" ref="name_paperformat"/>
+</record>
+
+</odoo>
```
Comment (vvrossem) [inline account_invoice_paid_transaction/views/report_name.xml]: to delete?

**Review by vvrossem — CHANGES_REQUESTED** [review]
I guess you don't override `_get_payment_vals` anymore. 
Some template to remove

**Comment by robinkeunen** [comment]
Indeed, I finally decided to separate the treatment of sale order related transactions from invoice related transactions. Thanks for the review and sorry for the dirty work I left behind :-)

**Review by vvrossem — APPROVED** [review]
LGTM

**Pull request merged**